.github/actions/validator_pypi_publish/add_build_prefix.py

```diff
@@ -0,0 +1,79 @@
+import re
+import sys
+
+
+def add_package_name_prefix(
+    pyproject_path, pep_503_new_package_name, validator_folder_name
+):
+    # Read the existing pyproject.toml file
```

Comment:
Given that we're doing all the regex matching and searching and swapping, could it make sense to just `pip install toml` and then do `content = toml.load(f)` so we don't have to do all the regex searching?  I know it's an extra dep, but between the re.subs and the defining section heads, perhaps it makes more sense to just treat it like we'd be treating a JSON file?

Reply:
funny thing is I had `toml` in there to read the name before and one of these AI tools must have swapped it out  at some point 😂 , but I avoided using toml to actually reconstruct the file as I saw some weird behavior I think with sections being added / dropped

Reply:
totally agree though, will add back in for the reading portions